cartesian_impedance_controller/src/cartesian_impedance_controller/cartesian_impedance_controller.cpp

```diff
@@ -15,6 +15,8 @@ bool CartImpedanceController::init(hardware_interface::PosVelEffJointInterface*
   m_controller_nh = controller_nh;
   m_hw = hw;
 
+  m_init_wrench = true;
```

Comment:
It is better to add a boolean parameter "zeroing_sensor_at_startup" to select if the reset is needed.